.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  tox:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
when don't we need fail fast?

Reply:
With `fail-fast` the whole job would be interrupted as soon as one of the subjobs fails. In my experience it's nicer to get both to run entirely to get all failures reported as opposed to only the first failure.